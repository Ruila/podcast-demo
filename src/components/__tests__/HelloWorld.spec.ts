import { describe, it, expect } from "vitest";
import sinon from "sinon";
import { mount } from "@vue/test-utils";
import Channel from "../channel/ChannelComponent.vue";
import { createTestingPinia } from "@pinia/testing";

describe("HelloWorld", () => {
  it("renders properly", () => {
    const wrapper = mount(Channel, {
      global: {
        plugins: [
          createTestingPinia({
            createSpy: sinon.spy,
          }),
        ],
      },
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
