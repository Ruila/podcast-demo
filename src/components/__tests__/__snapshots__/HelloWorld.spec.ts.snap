// Vitest Snapshot v1

exports[`HelloWorld > renders properly 1`] = `
<div
  class="flex justify-center"
>
  <div
    class="w-[768px] mb-[150px]"
  >
    
    
  </div>
</div>
`;
